riscv-runtime/src/std_support.rs

```diff
@@ -58,7 +58,7 @@ extern "C" fn sys_getenv(
     _varname: *const u8,
     _varname_len: usize,
 ) -> usize {
-    todo!()
+    0
```

Comment:
a small comment here would be nice though, saying this is implementing the empty env

Reply:
I'm fine after this fix.

Reply:
updated